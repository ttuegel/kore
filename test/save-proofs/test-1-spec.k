// Two claims, one passes and is saved to a file, the other one fails.
module VERIFICATION
  imports SAVE-PROOFS

  rule BB(X:Int) => CC(X:Int)
endmodule

module TEST-1-SPEC
  imports VERIFICATION

  rule BB(X:Int) => CC(X:Int)
  rule AA(X:Int) => DD(X:Int)
endmodule
