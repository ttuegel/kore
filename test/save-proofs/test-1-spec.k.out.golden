<k>
  AA ( X ) ~> DotVar1
</k>
